qa/pom.xml

```diff
@@ -33,38 +40,124 @@
     <dependency>
       <groupId>org.jboss.spec.javax.transaction</groupId>
       <artifactId>jboss-transaction-api_1.2_spec</artifactId>
-      <version>1.0.0.Final</version>
+      <version>${version.org.jboss.spec.javax.transaction}</version>
```

Comment:
Thanks @jmfinelli for the PR! Is this dependency still needed?  if yes why?

Reply:
TBH, I haven't checked if `jboss-transaction-api_1.2_spec` is still needed as the goal of this PR is to detach QA from both ArjunaJTS and Narayana-full. I guess we can remove this dependency (if not needed) during our migration to Jakarta. WDYT?

Reply:
I would presume to expect it is needed as https://github.com/jbosstm/narayana/pull/2049/files/55e3ae899f10e29e8ba50b9883bbc5df55e4435a#diff-98ba2d8741d0f457de06355f6c6951c198db3ca7d470aa35c97b861c6f1c2b94R115 only makes this dependency provided https://github.com/jbosstm/narayana/blob/master/ArjunaJTS/narayana-jts-idlj/pom.xml#L159 so I expect `qa/` needs to provide it.

Reply:
Thank you both! I agree with you @jmfinelli, the removal of the dependency can be done during the migration.